### DISCUSSION TITLE
Querying VictoriaMetrics with query_range needs the time shifted back 8 hours - is something misconfigured?

### DISCUSSION THREAD

**Discussion by fuhao009** [ccfos/nightingale#1351] [Q&A]
./victoria-metrics-prod  -loggerTimezone Asia/Shanghai -retentionPeriod=30d -storageDataPath=data
我插入数据后发现比我的数据要早8小时,比如我是
tt,test01=1i test03=1 1673055000000000000
tt,test01=1i test03=2 1673056000000000000
tt,test01=1i test03=3 1673057000000000000

实际查询需要http://127.0.0.1:8428/api/v1/query_range?start=2023-01-07T01:30:00.00Z&end=2023-01-07T01:31:00.00Z&query={__name__="tt_test03",test01="1i"}&step=1m

我的时间是早上9点30分查询需要-8小时,这个是为啥呀

**Comment by UlricQin** [comment]
展示的时候，返回的时间戳转换为时间的时候，需要指定时区，你如果不指定时区，可能用的UTC，你看看VM的UI里是不是有指定时区的地方
